sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="2003-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/libatomic_ops/releases/download/v$portVersion/libatomic_ops-$portVersion.tar.gz"
+CHECKSUM_SHA256="219724edad3d580d4d37b22e1d7cb52f0006d282d26a9b8681b560a625142ee6"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libatomic_ops$secondaryArchSuffix = $portVersion
+	lib:libatomic_ops$secondaryArchSuffix = 2.1.1 compat >= 1
+	lib:libatomic_ops_gpl$secondaryArchSuffix = 2.1.2 compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libatomic_ops$secondaryArchSuffix_devel = $portVersion
+	devel:libatomic_ops$secondaryArchSuffix = 2.1.1 compat >= 1
+	devel:libatomic_ops_gpl$secondaryArchSuffix = 2.1.2 compat >= 1
+	"
+REQUIRES_devel="
+	libatomic_ops$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gawk
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	runConfigure ./configure --disable-dependency-tracking \
+		--enable-shared \
+		--with-gnu-ld
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	fixPkgconfig
```

Comment:
* The `COPYING` file that gets installed in `$docDir` is almost the same as the `/system/data/licenses/GNU GPL v2` we have, so maybe we could replace it by a relative symlink.
* It might be a good idea to move the readme files to the `$developDocDir`.
* We like to drop the libtool files.
* We usually call `fixPkgconfig` right after `prepareInstalledDevelLibs`.

```diff
 	make install
 
-	fixPkgconfig
+	rm -f $docDir/COPYING
+	ln -r -s $dataDir/licenses/"GNU GPL v2" $docDir/COPYING
+	mkdir -p $developDocDir
+	mv $docDir/README* $developDocDir
+
+	# drop libtool files
+	rm -f $libDir/libatomic_ops*.la
 
 	prepareInstalledDevelLibs \
 		libatomic_ops \
 		libatomic_ops_gpl
+	fixPkgconfig
```

Reply:
@waddlesplash: Ok, thanks!

@ivmai: Here's the new diff, to address @waddlesplash's comment.
```diff
 	make install
 
-	fixPkgconfig
+	mkdir -p $developDocDir
+	mv $docDir/README* $developDocDir
+
+	# remove libtool files
+	rm -f $libDir/libatomic_ops*.la
 
 	prepareInstalledDevelLibs \
 		libatomic_ops \
 		libatomic_ops_gpl
+	fixPkgconfig
```
This is almost the same diff as in my previous comment. The only difference is that it does not replace the COPYING file by a symlink.

Reply:
You could just remove the file. The GNU GPL v2 doesn't require one to carry a copy of the license text with the package.